app/views/admin_only/member_app_waiting_reasons/edit.html.haml

```diff
@@ -0,0 +1,11 @@
+%header.entry-header
+  %h1.entry-title= t('.title', name_sv: @member_app_waiting_reason.name_sv, name_en: @member_app_waiting_reason.name_en  )
+
+.post-title-divider
+  %span
+
+.entry-content.wpcf7
+  = render 'form'
+
+.center
+  = link_to t('back'), admin_only_member_app_waiting_reasons_path, class: 'btn btn-default all-member-app-waiting-reasons', id: 'all-member-app-waiting-reasons'
```

Comment:
To keep this simple, since I don't think it will be used very much, I'd like to just change this to "back to the list of all reasons".